tests/observability/metrics/test_aaq_metrics.py

```diff
@@ -0,0 +1,147 @@
+import pytest
+from timeout_sampler import TimeoutSampler
+
+from tests.observability.metrics.utils import (
+    timestamp_to_seconds,
+    validate_metrics_value,
+)
+from utilities.constants import (
+    TIMEOUT_4MIN,
+    TIMEOUT_15SEC,
+)
+from utilities.virt import (
+    VirtualMachineForTests,
+    fedora_vm_body,
+    running_vm,
+)
+
+pytestmark = [
+    pytest.mark.usefixtures(
+        "enabled_aaq_in_hco_scope_package",
+        "updated_namespace_with_aaq_label",
+    ),
+]
+
+
+def parse_value(val):
```

Comment:
please give more meaningful name (what exactly is parsed and into what) + docstring